dev-util/lcov/lcov-1.13-r2.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/linux-test-project/lcov.git"
+	inherit git-r3
+else
+	SRC_URI="mirror://sourceforge/ltp/${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-linux ~x64-macos"
+fi
+
+inherit prefix
+
+DESCRIPTION="A graphical front-end for GCC's coverage testing tool gcov"
+HOMEPAGE="http://ltp.sourceforge.net/coverage/lcov.php"
+
+LICENSE="GPL-2+"
+SLOT="0"
+
+RDEPEND="
+	dev-lang/perl
+	dev-perl/GD[png]"
```

Comment:
Could you try making `GD[png]` an optional dependency that won't be pulled if a `png` USE flag is disabled for `lcov`? Besides `genpng` and the `-F` option to `genhtml`, it doesn't seem required but the current build always wants it.

Reply:
I've moved it into a `png` USE flag. I haven't noticed a difference, even in `genhtml` created files, so it'll be disabled by default.

Reply:
-9999 doesn't have this.

Reply:
Must have missed it the first time. Fixed now.